### PR TITLE
Add Application Access Action

### LINKED ISSUE #19: Application Access GitHub Action
Action for Application Access should output certificates for use with tools which support client certificates.

Proxying mode is out of scope for now, but can be introduced in future.

### PR DESCRIPTION
**Pull request by strideynet**
Closes #19 

```yaml
steps:
  - name: Install Teleport
    uses: teleport-actions/setup@v1
    with:
      version: 11.0.3
  - name: Fetch application credentials
    id: auth
    uses: teleport-actions/auth-application@v1
    with:
      # Specify the publically accessible address of your Teleport proxy.
      proxy: tele.example.com:443
      # Specify the name of the join token for your bot.
      token: my-github-join-token-name
      # Specify the length of time that the generated credentials should be
      # valid for. This is optional and defaults to "1h"
      certificate-ttl: 1h
      # Specify the name of the application you wish to access.
      app: grafana-example
  - name: Make request
    run: curl --cert {{ steps.auth.outputs.certificate-file }} --key {{ steps.auth.outputs.key-file }} https://grafana-example.tele.example.com/api/users
```

### CODE REVIEW

File: actions/auth-application/package.json
Comment (ryanclark) [inline actions/auth-application/package.json:3]: Should this be `1.0.0`?

**Review by ryanclark — APPROVED** [review]

File: actions/auth-application/package.json
Comment (strideynet) [reply actions/auth-application/package.json]: [0b8daa1](https://github.com/teleport-actions/root/pull/20/commits/0b8daa1cf5964f5b60b11c629e867860b25b3fb3)

**Comment by strideynet** [comment]
Holding off merging this until the new repository has been created pending approval in the GitHub Terraform repository.

**Comment by hungran** [comment]
Hi team @strideynet  how if I use `tbot` or `tsh proxy login` when using this?

**Pull request merged**
